AUTH48-reply.txt

```diff
@@ -0,0 +1,442 @@
+> below. Specifically, what is the second instance of "that" referring to?
+> Please review and let us know how this sentence should be
+> updated for clarity.
+> 
+> Original: 
+>    The group value that indicates that depends on what has been agreed
+>    with each operator ("operator-x-foo" vs. "operator-y" "bar"). -->
+> 
+> 
+> 12) <!-- [rfced] FYI - We have rearranged this sentence for readability. Please
+> let us know if this change is incorrect or if there are any objections.
+> 
+> Original: 
+>    Only when the zone was configured from a specific catalog zone, and
+>    the zone is removed as a member from that specific catalog zone, the zone and
+>    associated state (such as zone data and DNSSEC keys) MUST be removed from the
+>    consumer.
+> 
+> Current: 
+>    The zone and associated state (such as zone data and DNSSEC keys)
+>    MUST be removed from the consumer only when the zone was configured from a
+>    specific catalog zone and is removed as a member from that specific catalog
+>    zone. -->
+> 
+> 
+> 13) <!-- [rfced] FYI - We have updated the following text in Section 5.4
+> for easy readability. Specifically, we have rearranged the first
+> two phrases for better sentence flow and clarified some of the
+> text. Please let us know any objections.
+> 
+> Original: 
+>    When via a single update or transfer, the member node's label value
+>    (<unique-N>) changes, catalog consumers MUST process this as a member
+>    zone removal including all the zone's associated state (as described in
+>    Section 5.3), immediately followed by processing the member as a newly to
+>    be configured zone in the same catalog.  
+> 
+> Current: 
+>    When the member node's label value (<unique-N>) changes via a single update 
+>    or transfer, catalog consumers MUST process this as a member zone removal, 
+>    including the removal of all the zone's associated state (as described in 
+>    Section 5.3), and then immediately process the member as a newly added zone 
+>    to be configured in the same catalog.-->
+> 
+> 
+> 14) <!-- [rfced] FYI - We have updated the following text for readability. Please
+> let us know of any objections or technical inaccuracy.
+> 
+> Original:
+>    If in the process of a migration some consumers of the involved
+>    catalog zones did not catch the removal of the member zone from
+>    $OLDCATZ yet (because of a lost packet or downtime or otherwise), but
+>    did already see the update of $NEWCATZ, they may consider the update
+>    adding the member zone in $NEWCATZ to be a name clash (see
+>    Section 5.2) and as a consequence the member is not migrated to
+>    $NEWCATZ. 
+> 
+> Current:
+>    In the process of a migration, if some consumers of the involved
+>    catalog zones did not catch the removal of the member zone from
+>    $OLDCATZ yet (because of a lost packet, downtime, or otherwise) but
+>    already saw the update of $NEWCATZ, they may consider the update by

```

Comment:
indeed, maybe we can rearrange "the update adding the member zone" into "the adding-the-member-zone update" WDYT @peterthomassen @Habbie

Reply:
Yes. I would perhaps simplify this difficult sentence by removing the redundant "adding the member zone", like this: "...they may consider this update to be a name clash..."

Reply:
The update also appears earlier in the sentence, and when qualify the word "update" there, things get easier:

```
    If in the process of a migration some consumers of the involved
    catalog zones did not catch the removal of the member zone from
    $OLDCATZ yet (because of a lost packet or downtime or otherwise), but
    did already see the update of $NEWCATZ containing the addition of that
    member zone, they may consider this update to be a name clash (see
    Section 5.2) and as a consequence the member is not migrated to
    $NEWCATZ. 
```